hw/ip/dma/data/dma.hjson

```diff
@@ -355,7 +359,7 @@
       ]
     }
     { name: "CHUNK_DATA_SIZE"
-      desc: "Total size of the data blob involved in DMA movement."
+      desc: "Number of bytes to be transferred as a discrete unit in response to a single interrupt/firmware request."
```

Comment:
This was a bit confusing to me because I wasn't sure about the meaning of "as a discrete unit". Would the sentence still be correct as just "Number of bytes to be transferred in response to each interrupt/firmware request"?